iep-module-eureka/README.md

```diff
@@ -13,8 +13,16 @@ Guice module to configure [eureka](https://github.com/Netflix/eureka). It will d
 compile "com.netflix.iep:iep-module-eureka:${version_iep}"
 ```
 
-Internally at Netflix you will also need to pull in the config files from artifactory:
+Internally, at Netflix, you will also need to pull in the config files from Artifactory:
 
 ```
 compile "com.netflix.nfglue:nfglue-internal-configs:latest.release"
-```
\ No newline at end of file
+```
+
+Eureka gets its configuration from the Archaius 1 library. If you are using the standard
```

Comment:
Out of curiosity, have you guys looked at eureka-client-archaius2? It is entirely archaius2 based and does not instantiate any archaius1 classes at all.

Reply:
I looked at it a little bit, but never got around to trying it. One minor disappointment is that it would still pull archaius1 into the classpath even though it shouldn't get used. I would like to shed archaius1 entirely so that it is no longer discoverable and we don't get regressions because someones IDE found the old classes. I feel like it is a risk and our systems are harder to reason about when it is in the classpath but may or may not be setup and initialized. The current behavior means we notice pretty quickly if archaius1 is not initialized properly since we can remove it entirely.

Reply:
That's fair. The (complete) removal of archaius1 is on the TODO list, but it will involve some significant restructuring of the project so is not as trivial.